english/explore-outside/journey-to-flaming-tower/start-running/keep-running/keep-running.md

```diff
@@ -18,4 +20,4 @@ Now both of you:
 
 [Hail a graham cracker cab!](../../graham-cracker-cab/graham-cracker-cab.md)
 
-[Wait at the jellybus stop for a jellybus!](../../jellybus/jellybus.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.